.github/workflows/cocoapods-deploy.yml

```diff
@@ -0,0 +1,17 @@
+name: Deploy to Cocoapods
+on:
+  release:
+    types: [published]
```

Comment:
It looks like we _might_ need "prereleased" as well, the documentation isn't that clear.

Reply:
Yeah, I hate this documentation.. It gives a list of options but doesn't explain their meaning.